dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/persistence/dataset/DatasetVersion.java

```diff
@@ -584,7 +584,7 @@ public List<DatasetAuthor> getDatasetAuthors() {
                     DatasetField idType = e.get(DatasetFieldConstant.authorIdType);
                     author.setIdType(idType != null && !idType.getControlledVocabularyValues().isEmpty()
                             ? idType.getControlledVocabularyValues().get(0).getStrValue() : null);
-                    author.setIdValue(mapIfNotNull(e.get(DatasetFieldConstant.authorIdValue), DatasetField::getDisplayValue));
+                    author.setIdValue(mapIfNotNull(e.get(DatasetFieldConstant.authorIdValue), DatasetField::getValue));
```

Comment:
We could also add a `idDisplayValue` to `DatasetAuthor` to have it available when necessary.